test/Emitter/EmitterStackTest.php

```diff
@@ -110,16 +110,31 @@ public function testEmitLoopsThroughEmittersUntilOneReturnsNonFalseValue()
         $second->emit(Argument::type(ResponseInterface::class))
             ->willReturn(null);
 
-        $third = $this->prophesize(EmitterInterface::class);
-        $third->emit(Argument::type(ResponseInterface::class))
-            ->willReturn(false);
```

Comment:
Forgot this was a stack. The actual workflow is:

- Hits `$third`, which returns `false`.
- Hits `$second`, which returns `null`, and should halt execution of the stack.
- Should **not** hit `$first`.

So, the point is to note that the first non-`false` value will stop execution of the stack. We want to ensure that we can go more than one deep in the stack, and that it does not just halt on the first encountered.